pkg/postgresql/utils.go

```diff
@@ -263,6 +278,31 @@ func GetPGState(ctx context.Context, replConnParams ConnParams) (*cluster.Postgr
 	return nil, fmt.Errorf("query returned 0 rows")
 }
 
+func GetReplicationLag(ctx context.Context, replConnParams ConnParams) (uint, error) {
+	// Add "replication=1" connection option
+	replConnParams["replication"] = "1"
+	db, err := sql.Open("postgres", replConnParams.ConnString())
+	if err != nil {
+		return 0, err
+	}
+	defer db.Close()
+
+	rows, err := Query(ctx, db, "select pg_replication_lag()")
+	if err != nil {
```

Comment:
Sure, I know about it.
https://github.com/gravitational/stolon/pull/26/files#diff-d8eea19f0a5ff785d034ea580f183dccR294